### ISSUE TITLE
Javascript async function breaks parser

### ISSUE THREAD

**Issue by MichaelSchwab** [geany/geany#1933]
```javascript
async function test(a,b)
{
    console.log(a+b);
}

function test_missing(a,b)
{
    console.log(a+b);
}
```
As soon as one function is declared as async
the parser breaks and the symbollist does not show any more functions

Might be solved with
https://github.com/universal-ctags/ctags/pull/1544
but patching Release failed 1.33

**Comment by elextr** [comment]
Geany ctags parsers are a ways behind uctags upstream, so patches applying there may not apply here.  Updating is in progress, but is a lot of work and will take time.

**Comment by b4n** [comment]
Fixed by merging uctags parser in #2018.
